core/settings/qPrefCloudStorage.cpp

```diff
@@ -2,6 +2,8 @@
 #include "qPref.h"
 #include "qPrefPrivate.h"
 
+#include <QDebug>
+
```

Comment:
doesn't hurt, but I think once your messages are removed, we don't need it anymore